### DISCUSSION TITLE
github models access via api in open-webui?

### DISCUSSION THREAD

**Discussion by arifsha39** [open-webui/open-webui#6944] [General]
github models access via api in open-webui?

https://models.inference.ai.azure.com/

i added this and github token  the models show up in webui  select models options  but i get this error

Uh-oh! There was an issue connecting to gpt-4o-mini.
Unknown model: azureml://registries/azure-openai/models/gpt-4o-mini/versions/1

**Comment by VpgMackan** [comment]
[https://github.com/open-webui/open-webui/discussions/5461#discussioncomment-10758258](https://github.com/open-webui/open-webui/discussions/5461#discussioncomment-10758258)

This might help you. The error is due to azure using a difference way to send request.

**Reply by arifsha39** [reply]
> [#5461 (comment)](https://github.com/open-webui/open-webui/discussions/5461#discussioncomment-10758258)
> 
> This might help you. The error is due to azure using a difference way to send request.

thank you good sir!
